README.md

```diff
@@ -18,7 +18,10 @@ Requirements
 ------------
 
 - Python 2.7 or 3.2+
+- Less compilation requires the [less](http://lesscss.org/) package to be installed globally
+- SASS compilation requires [sass](http://sass-lang.com/install)
 
+Installation of these components is beyond the scope of this project.
```

Comment:
In my experience it prevents "Please add instructions for..." type requests as if we left something out or did an incomplete job.  This puts it clearly out of scope and heads that type of request off at the pass, as it were.  Again, your call.